### ISSUE TITLE
No clear instructions on where to place wikipedia file

### ISSUE THREAD

**Issue by iHnR** [dps/remarkable-wikipedia#24]
It's unclear to me where to place the wikipedia.zim file on the remarkable. I've tried a few different locations but it's not recognized. The remarkable2 does not support usb OTG on the main port so using a USB isn't possible anyway. The install instructions mention that you can put a small wikipedia on the device, but fail to mention where.

**Comment by qubist** [comment]
Agreed. If I install this app through `toltec`, where does it look for .zim files?
